<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAddAccount</class>
 <widget class="QDialog" name="CAddAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>132</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>393</width>
    <height>132</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>393</width>
    <height>132</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加賬號</string>
  </property>
  <widget class="QComboBox" name="comboBox_area">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>觉醒内测区</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>合作体验区</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_server">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>觉醒内测1服</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>觉醒内测2服</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>合作体验服</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>30</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 127);</string>
   </property>
   <property name="text">
    <string>大区:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 0, 127);</string>
   </property>
   <property name="text">
    <string>服务器:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_enter_acc">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确  定</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>账号:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_acc"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>密码:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_psw"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>30</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 0);</string>
   </property>
   <property name="text">
    <string>脚本:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_script">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_enter_acc</sender>
   <signal>clicked()</signal>
   <receiver>CAddAccount</receiver>
   <slot>PushButton_Enter_Acc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>PushButton_Enter_Acc()</slot>
 </slots>
</ui>
